src/components/UI/TransferLogContainer/TransferLogContainer.js

```diff
@@ -12,10 +12,10 @@ import {
 } from './TransferLogContainer.strings';
 
 export const TransferLogContainer = ({transferIndex, children}) => {
-  const [showChildren, setShowChildren] = useState(false);
+  const [showChildren, setShowChildren] = useState(transferIndex > 1);
 
   useEffect(() => {
-    setShowChildren((!!transferIndex || transferIndex === 0) && transferIndex > -1);
+    setShowChildren(transferIndex > -1);
```

Comment:
Why do we need it?

Reply:
without it, the state won't update correctly. Eventually, lead to a scenario that a click on a link will not expand the transfer log.